mypy.ini

```diff
@@ -0,0 +1,2 @@
+[mypy]
```

Comment:
You may want to add the following to this:

```
show_error_codes = true
python_version = 3.9
```

`show_error_codes` allow supressions to mark the specific error code from `mypy` such as `# ignore[misc]`

`python_version` makes sure that the typings are consistent between `mypy` users. If one users has Python 3.8 and one Python 3.9 they will receive the same `mypy` results.

Reply:
updated